### DISCUSSION TITLE
Prompt to setup password upon creation

### DISCUSSION THREAD

**Discussion by trex7891** [oidc-wp/openid-connect-generic#343] [General]
Hello there, thanks for the plugin...
Could you please point me to the right direction, I want to prompt user registered from here, to setup their password upon successfully create the user.
Thanks a lot....

**Comment by timnolte** [comment]
@trex7891 so I'm not clear on what password you are looking to have users set. The WordPress account password is never used when using SSO. Password prompts and change requests would be completely controlled by your IDP.

**Comment by trex7891** [comment]
@timnolte sorry for being unclear. I am trying to make it optional to login using password. I see at line 931 of client-wrapper that the password is generated anyway using wp_generate_password. I want to save the need of changing password through reset.

**Comment by trex7891** [comment]
@timnolte Hello again Tim, I think this question can be disregarded. 
To be very frank, I was evaluating it and assessing how close/far is it to be used for a 'registration to retrieve information from 3rd party provider through oAuth2', so, my purpose is for a 1 time retrieval and WP user creation, not SSO.
Thanks, Tim
